lib/collection/src/operations/cluster_ops.rs

```diff
@@ -47,6 +47,10 @@ pub enum ClusterOperations {
     AbortResharding(AbortReshardingOperation),
     /// Trigger replication of points between two shards
     ReplicatePoints(ReplicatePointsOperation),
+
+    /// Introduce artificial delay to a node
+    #[cfg(feature = "staging")]
+    SlowDownNode(SlowDownNodeOperation),
```

Comment:
Nice!

Nit: I suggest to prefix these with 'test', so that they are clearly distinguishable from regular operations even in places where we call them. So it might become `TestSlowDown` and `test_slow_down`.

Wdyt?